tests/coverage/test_gcovr.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+##############################################################
+# Folders structure (28.07.2020)
+# NOTE: Drawing will be removed after approval script
+##############################################################
+# facelift/                 <-- directory with source code
+# ├─ build/
+# │  ├─                     <-- working directory
+# │  └─
+# ├─ ci/
+# │  ├─ appveyor
+# │  └─ travis
+# ├─ cmake/
+# ├─ CMakeLists.txt
+# ├─ src/
+# ├─ tests/
+# │  ├─ asyncfunctions/
+# │  ├─ benchmarking/
+# │  ├─ cmake/
+# │  ├─ CMakeLists.txt
+# │  ├─ combined/
+# │  ├─ coverage/
+# │  │   ├─ read.me
+# │  │   └─ test_gcov.sh*   <--- executable script
+
+function outerdir(){
```

Comment:
Did you reimplement the dirname command ?

Reply:
Please take attention to difference of output paths for two cases.
Just for example:
```
echo "output #1: " $(cd "dirname $0"; pwd -P)
case #1:  /home/user/Workspace/facelift/build
echo "output #2: " $DIR_SRC
case #2:  /home/user/Workspace/facelift
```

Note:
case # 1: needs additional processing to get the path to the source code
case # 2: the path to the source code (outer folder) no matter how deep in the subfolders the script is placed.